common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -654,8 +731,7 @@ public MergeStatuses finalizeShuffleMerge(FinalizeShuffleMerge msg) {
           // If no blocks pushed for the finalizeShuffleMerge shuffleMergeId then return
           // empty MergeStatuses but cleanup the older shuffleMergeId files.
           submitCleanupTask(() ->
-              closeAndDeleteOutdatedPartitions(
-                  appAttemptShuffleMergeId, mergePartitionsInfo.shuffleMergePartitions));
+              deleteCurrentShufflePartitions(appShuffleInfo, shuffleId, appAttemptShuffleMergeId));
```

Comment:
I checked the `appAttemptShuffleMergeId` in the code before. 
I think if we want to delete partitions merged data, then we must delete the corresponding ShuffleMergeId in DB (Otherwise, inconsistency will occur when restoring shuffle info from db and shufflemergeid is only generated when finalized)

Reply:
And I think when deleting partition information, we may don’t store shuffleMergeID in DB anymore